exotica_core/include/exotica_core/kinematic_tree.h

```diff
@@ -145,9 +154,16 @@ class KinematicTree : public Uncopyable
     const Eigen::MatrixXd& GetJointLimits() const { return joint_limits_; }
     void SetJointLimitsLower(Eigen::VectorXdRefConst lower_in);
     void SetJointLimitsUpper(Eigen::VectorXdRefConst upper_in);
+    void SetJointLimitsVelocity(Eigen::VectorXdRefConst velocity_in);
+    void SetJointLimitsAcceleration(Eigen::VectorXdRefConst acceleration_in);
+    void SetFloatingBaseLimitsPosXYZEulerZYX(const std::vector<double>& lower, const std::vector<double>& upper, const std::vector<double>& velocity, const std::vector<double>& acceleration);
+    void SetPlanarBaseLimitsPosXYEulerZ(const std::vector<double>& lower, const std::vector<double>& upper, const std::vector<double>& velocity, const std::vector<double>& acceleration);
     void SetFloatingBaseLimitsPosXYZEulerZYX(const std::vector<double>& lower, const std::vector<double>& upper);
     void SetPlanarBaseLimitsPosXYEulerZ(const std::vector<double>& lower, const std::vector<double>& upper);
     std::map<std::string, std::vector<double>> GetUsedJointLimits() const;
+    bool GetHasAccelerationLimit() const { return has_acceleration_limit; }
```

Comment:
Google Code Style says if it's a pure getter/setter, use `set_{name of variable}` and `get_{name of variable}`. We haven't honoured this everywhere yet, but I am using it for everything new I am adding and then will carry it through once I touch APIs.

Reply:
All functions are capitlisaed in the code - so should it be Get_{Name of variable}, or get_{name of variable}?

Reply:
ie: get_HasAccelerationLimit() or Get_HasAccelerationLimit() or get_has_acceleration_limit()?

Reply:
`HasAccelerationLimit` as resolved via WhatsApp

Reply:
Reasoning behind this is:
If the parameter requires a getter/setter, especially they are not trivial, use Get/Set prefix.
If the parameter is trivial and starts with a verb (Is/Has) don't add another prefix to improve readability.

Also, mixing underscores and camel casing is not good.